### PR TITLE
Added direct lvm recommendation

### PR DESCRIPTION
**Pull request by joel-hamill**
## What are your changes?

Added direct lvm recommendation
## What type of changes does your doc introduce?
- [ ] Bug fix
- [x] New feature 
## What is the urgency level of this change?
- [ ] Blocker
- [ ] High
- [x] Medium
## I have completed these items:
- [x] I have tested all commands and code snippets.
- [ ] I have built locally and tested for broken links and formatting.
- [x] I have made this change for all affected versions (e.g. 1.7, 1.8, and 1.9).
- [x] My doc follows the style guide: https://github.com/dcos/dcos-docs#contributing.

Ping @emanic @sascala @joel-hamill for reviewing pull request changes.

### CODE REVIEW

File: 1.7/administration/installing/custom/system-requirements/install-docker-centos.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ In addition to the general [Docker requirements and recommendations for DC/OS][1
 
 * Format node storage as XFS. As of CentOS 7.2, "<a href="https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/7.2_Release_Notes/technology-preview-file_systems.html" target="_blank">only XFS is currently supported for use as a lower layer file system</a>".
 
+* Use `direct-lvm` as the method to access Docker backing store in production. For more information, see [Selecting a Storage Driver](https://docs.docker.com/engine/userguide/storagedriver/selectadriver/).
```
Comment (cmaloney) [inline 1.7/administration/installing/custom/system-requirements/install-docker-centos.md]: This conflicts with the "Format node storage as XFS" and "overlayfs" instructions throughout the rest of this.

**Review by cmaloney — CHANGES_REQUESTED** [review]

File: 1.7/administration/installing/custom/system-requirements/install-docker-centos.md (hunk 1 above)
Comment (joel-hamill) [inline 1.7/administration/installing/custom/system-requirements/install-docker-centos.md]: @cmaloney so should this line not be added, or should we revise the rest of the doc?

Comment (cmaloney) [inline 1.7/administration/installing/custom/system-requirements/install-docker-centos.md]: We should document Direct-LVM as the most production ready / stable on CentOS. That requires that the rest of this document be re-written / re-worked (Just adding a bullet makes the documentation inconsistent with itself).

Comment (joel-hamill) [inline 1.7/administration/installing/custom/system-requirements/install-docker-centos.md]: @karlkfi ^^

Comment (karlkfi) [inline 1.7/administration/installing/custom/system-requirements/install-docker-centos.md]: Sounds reasonable to me, but I haven't tested Direct-LVM, don't know why we're recommending it, and don't know if it works with docker-in-docker (aka Jenkins). If eng wants to change the recommendation then they should change the docs to reflect that.

**Comment by joel-hamill** [comment]
@cmaloney @karlkfi Okay, if we don't want to change that's fine. I was working off this internal JIRA from support: https://mesosphere.atlassian.net/browse/DOCS-1052.

**Comment by karlkfi** [comment]
@joel-hamill: changing the recommendation in not something that should be taken lightly. We need buy-in from engineering that they are testing this in CI and support that they are willing to support it going forward. Just because Docker recommends it, doesn't mean we need to blindly recommend the same. And if we do want to recommend it then we need proof (manual and automated testing) that it passes all known use cases.

I'm not saying we don't want to change it. I'm saying there is a cost to changing it and we haven't paid it yet or even promised to pay it.

I would like to see buy-in from @spahl & @graham-murphy before we change the recommendations. And when we do, we need to change it everywhere.

**Pull request closed without merging**